addons/website_sale/controllers/main.py

```diff
@@ -4,20 +4,23 @@
 import json
 import logging
 from datetime import datetime
+
+from psycopg2.errors import LockNotAvailable
```

Comment:
`psycopg2.errors` is not part of psycopg 2.7.7 which is the version of psycopg used with debian buster, an OS that is officially supported with odoo 16.0

Reply:
See also #106345, I'll fix the problem.

Reply:
PR that fixes the problem: https://github.com/odoo/odoo/pull/208770

Reply:
Indeed, thanks for pointing out the issues and making a PR!